internal/services/network/ip_group_resource.go

```diff
@@ -47,8 +47,10 @@ func resourceIpGroup() *pluginsdk.Resource {
 			"resource_group_name": azure.SchemaResourceGroupName(),
 
 			"cidrs": {
-				Type:     pluginsdk.TypeSet,
-				Optional: true,
+				Type:       pluginsdk.TypeSet,
+				ConfigMode: pluginsdk.SchemaConfigModeAttr,
+				Optional:   true,
+				Computed:   true,
```

Comment:
unfortunately this'd be a breaking configuration change, instead we'll need to revert this:

```suggestion
				Optional:   true,
```

and instead split the Create and Update methods here such that we retrieve the existing value for `cidrs` from the API and reuse this during the update, unless the `cidrs` block has changed, for example:

```
existing, err := client.Get(...)
if err != nil { .. }

update.Properties.Cidrs = existing.Properties.Cidrs
if d.HasChange("cidrs") {
    update.Properties.Cidrs = expandCidrs(d.Get("cidrs").([]interface{})
}
```

which'd allow users to make use of `ignore_changes` to ignore changes to the `cidrs` block on this resource, and use the separate resource instead - which'd need to be called out in the documentation for both of these resources